swap/src/database.rs

```diff
@@ -126,9 +127,12 @@ pub async fn open_db(
                 sqlite.insert_peer_id(swap_id, peer_id).await?;
             }
 
+            tracing::info!("Sucessfully migrated data to sqlite! Using sqlite.");
+
             Ok(Arc::new(sqlite))
         }
         (_, _, false) => {
+            tracing::debug!("Sled database not found. Using sqlite.");
```

Comment:
I know it is explicit from the previous pattern but having `(false, _, false)` here would be more readable I think.